cli/tests/integration/run_tests.rs

```diff
@@ -2334,7 +2334,7 @@ fn issue9750() {
       Input("yy\n"),
       Output("⚠️  ️Deno requests env access. Run again with --allow-env to bypass this prompt.\r\n   Allow? [y/n (y = yes allow, n = no deny)]"),
       Input("n\n"),
-      Output("⚠️  ️Deno requests env access to \"SECRET\". Run again with --allow-run to bypass this prompt.\r\n   Allow? [y/n (y = yes allow, n = no deny)]"),
+      Output("⚠️  ️Deno requests env access to \"SECRET\". Run again with --allow-env to bypass this prompt.\r\n   Allow? [y/n (y = yes allow, n = no deny)]"),
```

Comment:
Ummm... how is this passing on linux & mac?

Reply:
This fails for me on linux and then hangs forever.

Reply:
See #14014, couldn't figure out why CI wasn't failing but was hanging locally, seems you figured it out.